packages/math-input/src/components/keypad/mobile-keypad-internals.tsx

```diff
@@ -220,6 +220,7 @@ class MobileKeypadInternals
                                 containerWidth > expandedViewThreshold
                             }
                             showDismiss
+                            scientific={keypadConfig?.scientific}
```

Comment:
Small nit, wondering if this should be up with the rest of the button sets (even though it's triggered differently). Also, does this need to be `scientific={isExpression && keypadConfig?.scientific}`? Or do we expect this to be used on the fraction keypad too?

Reply:
Since we weren't able to get full requirements from content, I'm not sure if they will ever have a use case of scientific + fraction, this question would definitely deserve some time to flesh out if we decide to implement.